fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -781,6 +789,33 @@ public void executeByLegacy(TUniqueId queryId) throws Exception {
         }
     }
 
+    private void syncJournalIfNeeded() throws Exception {
+        if (!context.getSessionVariable().enableStrongConsistencyRead
+                || !Config.enable_strong_consistency_read) {
```

Comment:
do not need this config, I think session variable is ok. Because if user want to affect all users, it could use set global session variable = xxx.